packages/components/select/src/useSelect.ts

```diff
@@ -345,6 +345,7 @@ export const useSelect = (props, states: States, ctx) => {
       }
       const inputs = selectWrapper.value?.querySelectorAll('input') || []
       if (
+        !(props.remote || props.filterable) ||
```

Comment:
Perhaps it would be better to change the judgment condition to the following:
```suggestion
        (!props.filterable && !props.defaultFirstOption && !isUndefined(props.modelValue)) ||
```

Reply:
done~